Servers/UI/OJS.Servers.Ui/ClientApp/src/identity-config.ts

```diff
@@ -3,13 +3,15 @@ import { WebStorageStateStore } from 'oidc-client';
 const appBaseUrl = window.URLS.UI_URL;
 const authorityBaseUrl = process.env.REACT_APP_PLATFORM_IDENTITY_BASE_URL;
 const authorityUrl = `${process.env.REACT_APP_PLATFORM_IDENTITY_BASE_URL}${process.env.REACT_APP_PLATFORM_IDENTITY_AUTHORITY_URL_POSTFIX}`;
+const authorityRegisterUrl = `${authorityUrl}/register`;
 // eslint-disable-next-line max-len
 
 const IDENTITY_CONFIG = {
     authority: authorityBaseUrl,
     client_id: process.env.REACT_APP_IDENTITY_CLIENT_ID,
     redirect_uri: `${appBaseUrl}/logincallback`,
     login: `${authorityUrl}`,
+    register: authorityRegisterUrl,
```

Comment:
This should not be here. Move it to `urls`. This file is mostly redundant and is on its way to deletion.

Reply:
I suggest i log this into an issue as there are changes needed in the index.cshtml file for this. We might log a bigger issue to remove this file and fix this in it.

Reply:
Don't remove it now. Just move the `url`